packaging/deb/rules

```diff
@@ -18,7 +18,7 @@ override_dh_auto_configure:
 
 override_dh_auto_build:
 	env
```

Comment:
I noticed we still have this `env` call here for debugging. Happy to keep or remove, I just noticed that we don't _need_ it anymore really.

Reply:
I think it's fine to remove.